lib/spack/docs/tutorial_modules.rst

```diff
@@ -349,7 +631,7 @@ name. We have the possibility to add suffixes to differentiate them:
       whitelist:
         -  gcc
       blacklist:
-        -  '%gcc@4.8'
+        -  '%gcc@5.4.0'
```

Comment:
A couple of lines above this one (but not currently not part of the PR) is the sentence "We have the possibility to add suffixes...".  A better fit for the rest of the tone is "We can add suffixes...".